packages/elements/src/email-field/index.ts

```diff
@@ -70,4 +71,15 @@ export class EmailField extends TextField {
       'multiple': this.multiple
     };
   }
+
+  /**
+   * Check if input should be re-validated
+   * @param changedProperties Properties that has changed
+   * @returns True if input should be re-validated
+   */
+  protected shouldValidateInput (changedProperties: PropertyValues): boolean {
+    return changedProperties.has('value')
+      || changedProperties.has('pattern')
+      || changedProperties.has('minlength');
```

Comment:
should include `multiple` and `maxlength`

Reply:
Added